devtools/test_dashboard/server.js

```diff
@@ -72,7 +72,7 @@ function readFiles(files) {
 }
 
 function createMocksList(files) {
-    var mocksList = files.map(function(file) {
+    var mocksList = files.filter(function(file) {return file.extname === '.json';}).map(function(file) {
```

Comment:
Nice find!

Can we make this filter clearer though? Something like:

```js
// in case pollutants (e.g .DS_Store) got in the mock directory
var jsonFiles = files.filter( /* */ );
  
var mockList = jsonFiles.map(function(mock) { /* */ })
```